app/routes/abstract-module-route.js

```diff
@@ -1,6 +1,8 @@
-import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
 import Ember from 'ember';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
```

Comment:
@billybonks I took a quick look at our code and it looks like its about half and half as to the Ember import being at the top of the file or alphabetically, so there really isn't a consistent standard.  I'm fine moving along this PR as is.